app/models/user.rb

```diff
@@ -1,6 +1,7 @@
 class User < ActiveRecord::Base
 
-  validate :first_name, :last_name, :email, :novell_username, :webid
+  validate :email, :novell_username, :webid
+  validates_presence_of :first_name, :last_name
```

Comment:
you mean changing line 4 into two lines like this:
```rspec
validates :first_name, presence: true
validates :last_name, presence: true
```
and leave line 3 as it is?